test/suites/core/non_secure/core_ns_positive_testsuite.c

```diff
@@ -20,7 +20,6 @@
 #include "tfm_veneers.h"
 #endif /* TFM_PSA_API */
 #ifdef TFM_ENABLE_IRQ_TEST
-#include "platform_irq.h"
```

Comment:
Well, this is not clean solution @oyvindronningstad 
I am afraid this can't be accepted upstream.
Maybe bring back an empty platform_irq.h for nRF targets (I remember I had added that during porting and was asked to remove it as not used)

Reply:
It's not really clean to include platform_irq.h here because it contains platform-specific stuff. And anyways, it's not used in this file.

Reply:
In these cases, I would normally want this to be sent upstream as a separate PR, first. We're not doing this now, but I assume we have a plan to do this :)

Reply:
Of course. Can we assume that this is fine, and if they don't accept it upstream when we PR it, we add an empty platform_irq.h at that point?

Reply:
sounds good